src/main/java/org/jabref/gui/maintable/MainTable.java

```diff
@@ -112,6 +113,15 @@ public MainTable(MainTableDataModel model, JabRefFrame frame,
         // Enable sorting
         model.getEntriesFilteredAndSorted().comparatorProperty().bind(this.comparatorProperty());
 
+        model.getEntriesFilteredAndSorted().addListener(
+            (ListChangeListener<BibEntryTableViewModel>) change -> {
+                if (change.next()) {
+                    if (change.wasAdded()) {
```

Comment:
Could you please also add a `!change.getAddedSubList().isEmpty()` check to make sure that the code below executes without exception (that test might be covered by `wasAdded` but I'm not sure)

Reply:
It's covered by `wasAdded()`.